### PR TITLE
add user references and logo for PTG

### PR DESCRIPTION
**Pull request by agnes512**
Add user reference for [PTG pitagorines group](https://www.pitagorinesgroup.com/en/)

references:
[Add user references and logos](https://github.com/strongloop/loopback.io/pull/857)
[Looking for LoopBack user references](https://github.com/strongloop/loopback-next/issues/3047)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -365,6 +371,17 @@ <h3 class="section-heading">What our users say</h3>
...
+            </div>  
+            <br>
+            <br>
+            <div class="row text-center" style="justify-content: right">
+                <div class="col-md-6" style="text-align: left; padding-right:60px">
+                    "Our company has been using Loopback for several years. We started using it from its beginnings and the truth is that we love this new version. We want to congratulate the Strongloop Team from PTG Team for making developer's life a bit easier."
+                    <br>
+                    <br>
+                    <br>
+                    <b>- Jesús Mora, CTO & Co-Founder</b>
```
Comment (dhmlau) [inline index.html]: To be consistent, please add the company name for the title.
```
Jesús Mora, CTO & Co-Founder, Pitagorines Group
```

**Comment by agnes512** [comment]
After adding the logo:
Screen Shot 2019-08-06 at 11 43 48 AM

After adding the testimony:
Screen Shot 2019-08-06 at 11 45 23 AM

File: index.html [hunk 2]
```diff
@@ -365,6 +371,17 @@ <h3 class="section-heading">What our users say</h3>
                 <br>
                 <b>- Benjamin Boudreau, FoKo Software Developer</b>
                 </div>
+            </div>  
+            <br>
+            <br>
+            <div class="row text-center" style="justify-content: right">
+                <div class="col-md-6" style="text-align: left; padding-right:60px">
+                    "Our company has been using Loopback for several years. We started using it from its beginnings and the truth is that we love this new version. We want to congratulate the Strongloop Team from PTG Team for making developer's life a bit easier."
```
Comment (dhmlau) [inline index.html]: `Loopback` -> `LoopBack`

**Review by dhmlau — COMMENTED** [review]
@agnes512, thanks for updating this.  If you have a screen cap of what's that section is gonna look like, that would be easier to review. Thanks.

**Review by dhmlau — APPROVED** [review]
LGTM. Please squash the commits before merging. Thanks.

**Pull request merged**